validate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Validate AppStream catalog.
+
+set -e
+
+PROGDIR=$(dirname "$0")
+cd "$PROGDIR"
+
+# Ideally we'd use appstreamcli, but the version in EOS is currently too
+# old and has bugs with merge="append" entries.
```

Comment:
Do we have a task open about moving from `appstream-util` to `appstreamcli`? We probably should, since `appstream-util` is now basically unmaintained.

I found https://phabricator.endlessm.com/T28608 when I looked. It’s related.

Reply:
We should. I'll open one shortly. It should be straightforward to import the version from bookworm.

Reply:
https://phabricator.endlessm.com/T34591

Reply:
Nitpick: I’d add `# FIXME: https://phabricator.endlessm.com/T34591` to the comment so that finding it later is easier (with `git grep FIXME`), and checking up on the task from the comment is easier.

LGTM to merge either way though